fem/bilininteg.hpp

```diff
@@ -2935,7 +2935,7 @@ class VectorDivergenceIntegrator : public BilinearFormIntegrator
    Vector pa_data;
    const DofToQuad *trial_maps, *test_maps; ///< Not owned
    const GeometricFactors *geom;            ///< Not owned
-   int dim, ne, nq;
+   int dim, sdim, ne, nq;
```

Comment:
I don't think anything needs to be done, but it's worth considering whether `sdim` should be a member variable, given I believe it's a work variable that can be instantiated in `VectorDivergenceIntegrator::AssembleElementMatrix2` without noticeable cost.

On the other hand, the same can be asked about the other `int`s on this line (are `ne` and `nq` even used anywhere?)... so I can see keeping `sdim` here to be consistent.